### PR TITLE
Proofreading and fix cross-document link

### PR DESCRIPTION
**Pull request by dimhotepus**
* Fix typos.
* Fix small grammar mistakes.
* Fix link to `std::make_unique`, as it is absolute one.

### CODE REVIEW

**Comment by AnthonyCalandra** [comment]
Hi, change looks good. Can you also make the same edit in README.md? Thanks!

**Review by AnthonyCalandra — CHANGES_REQUESTED** [review]
Make the same change to README.md.

**Review by AnthonyCalandra — CHANGES_REQUESTED** [review]
Thanks for submitting all these changes! They were very good.

Since I don't want top post a comment for every instance of this error I'll post here: the changes where you change "an" to "a" before words like "rvalue" are actually incorrect changes. The reason for this is because you don't use "a" before words that have vowel sounds at the beginning of their pronunciation. For example, "rvalue" sounds like "are value" when you say it aloud, and since "are" begins with a vowel it should be proceeded by "an". See [here](https://www.quora.com/Which-is-correct-in-English-an-R-or-a-R) or [here](https://csgillespie.wordpress.com/2011/03/26/an-r-package-or-a-r-package/). Same reasoning applies to "lvalue" because it's pronounced as "elle value".

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -11,8 +11,8 @@ I'm not very picky about how you should contribute, but I ask that the following
...
+* Make sure you have also added the feature to the separate major C++ readme files (i.e. CPP11.md, CPP14.md, etc.).
 * Keep additions/deletions of content consistent with the cheatsheet's goals.
 
 ## Goals
-My goal for this cheatsheet is to prefer conciseness over absolute completeness. Examples of features should be minimal: if an example is overly complicated, large, or is more of an obscure usage of the feature then it will most likely be rejected in review. The reason for this goal is to teach users what the most popular uses of these features will be, and for a more thorough investigation, to learn about those from external C++ resources.
+My goal for this cheatsheet is to prefer conciseness to absolute completeness. Examples of features should be minimal: if an example is overly complicated, large, or is more of an obscure usage of the feature then it will most likely be rejected in review. The reason for this goal is to teach users what the most popular uses of these features will be, and for a more thorough investigation, to learn about those from external C++ resources.
```
Comment (AnthonyCalandra) [inline CONTRIBUTING.md:18]: The original English is correct here.

Comment (dimhotepus) [reply CONTRIBUTING.md]: Good to know, thanks! Reverted such changes.

**Comment by dimhotepus** [comment]
> Thanks for submitting all these changes! They were very good.
> 
> Since I don't want top post a comment for every instance of this error I'll post here: the changes where you change "an" to "a" before words like "rvalue" are actually incorrect changes. The reason for this is because you don't use "a" before words that have vowel sounds at the beginning of their pronunciation. For example, "rvalue" sounds like "are value" when you say it aloud, and since "are" begins with a vowel it should be proceeded by "an". See [here](https://www.quora.com/Which-is-correct-in-English-an-R-or-a-R) or [here](https://csgillespie.wordpress.com/2011/03/26/an-r-package-or-a-r-package/). Same reasoning applies to "lvalue" because it's pronounced as "elle value".

Fixed. Today I definitely learned something new.

**Review by AnthonyCalandra — APPROVED** [review]
Looks good to me. Thanks!

**Pull request merged**